homework-submission/b-jason-week1hw/src/Header.js

```diff
@@ -0,0 +1,19 @@
+import React, { Component } from 'react';
+
+
+export default class Header extends Component {
+
+
+    render() {
+
+        return (
+
+            <header className="App-header" >
+
+                <h1><strong>ToDoList App</strong></h1 >
+
+            </header >
+
+        )
+    }
+} 
```

Comment:
### .eslintrc
this icon means no new line at the end of the file, it's against the rules, look Line [38](https://github.com/thwjason/React/blob/jason-3-weekHW/.eslintrc#L38) so keep a new line on every end of file